fusillade/config.py

```diff
@@ -12,6 +12,7 @@ class Config:
     def get_admin_emails(cls):
         if not cls._admin_emails:
             cls._admin_emails = [admin.strip() for admin in os.environ['FUS_ADMIN_EMAILS'].split(',') if admin.strip()]
+            assert cls._admin_emails, "initial administrator must be specified."
```

Comment:
Could the message contain info about setting `FUS_ADMIN_EMAILS`?

Reply:
Fixed